docs/siem/reference/timeline-schema.asciidoc

```diff
@@ -0,0 +1,192 @@
+[[timeline-object-schema]]
+[role="xpack"]
+== Timeline object schema
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|`columns` |<<col-obj, columns[]>> |The timeline columns displayed in the UI.
+|`created` |Float |The time the timeline was created, using a 13-digit Epoch
+timestamp.
+|`createdBy` |String |The user who created the timeline.
+|`dataProviders` |<<dataProvider-obj, dataProviders[]>> |The dropzone query
+that determines which events are displayed in the timeline.???
+|`dateRange` |Object |The timeline's range.
+|`description` |String |The timeline's description.
+|`eventNotes` |<<eventNotes-obj, eventNotes[]>> |Notes added to specific
+events in the timeline.
+|`eventType` |String a|Event types displayed in the timeline, which can be:
+
+* `all`: all events
+* `raw`: raw events only
+* `signal`: signals only
+
+|`favorite` |<<favorite-obj, favorite[]>> |Indicates who marked a timeline as a
+favorite, and at what time.
+|`filters` |<<filters-obj, filters[]>> |Filters used in addition to the
+dropzone query. ???
+|`globalNotes` |<<globalNotes-obj, Object[]>> |Notes added to the timeline.
+|`kqlMode` |String a|Indicates whether the KQL bar filters the dropzone query
+results or searches for additional results, where:
+
+* `filter`: filters dropzone query results
+* `search`: displays additional search results
+
+|`kqlQuery` |<<kqlQuery-obj, kqlQuery>> |Object describing the KQL bar query.
+|`pinnedEventIds` |pinnedEventIds[] |Object containing the IDs of pinned
+events.
+|`savedObjectId` |String |The timeline's saved object ID.
+|`savedQueryId` |String |If used, the saved query ID used to filter or search
+dropzone query results.
+|`sort` |sort a|Object indicating how rows are sorted in the timeline's grid:
+
+* `columnId` (string): The ID of the column used to sort results.
+* `sortDirection` (string): The sort direction, which can be either `desc` or
+`asc`.
+
+|`status` |String |: ???
```

Comment:
The acceptable value would be active, immutable, and draft. But we don't want user to touch this field, and we don't expose this when exporting timelines, guess we could skip this field here.